lib/msal-browser/docs/logging.md

```diff
@@ -59,5 +59,24 @@ An example usage in a sample can be accessed [here](https://github.com/AzureAD/m
 
     ![browser console](./images/BrowserLogEnablement.png)
 
+## Override log level and PII setting
+
+These are the steps to override MSAL log level and PII settings to troubleshoot errors in non-dev environments:
+
+### Navigate to session storage
+
+1. Open developer tools by pressing F12
+2. Navigate to `Application` tab
```

Comment:
These instructions vary per browser.  For instance, Firefox is Developer Tools -> Storage Tab...  The Application tab isn't needed here.  Can you update these to be called out per browser.

Reply:
That's a very good call, thank you. Updated docs.